lib/filter_lib/tests/test_pagination.py

```diff
@@ -7,7 +7,9 @@
     make_pagination,
     paginate,
 )
+from ..src.query_modificator import form_query
```

Comment:
Avoid using relative imports, if it's possible

Reply:
All imports in library itself and in tests for it are written in a relative manner.
I would prefer to keep all imports this way in order not to break something unintentionally

Reply:
All relative imports are rewritten to absolute